src/blog/2019-04-30-hub-enhancements-apr-2019/index.md

```diff
@@ -0,0 +1,44 @@
+---
+extends: blog/_post.html
+title: Stencila Hub Enhancements for April 2019
+author: Ben Shaw
+date: 30 April 2019
+abstract: |
+    We're always working to improve the user experience on the Stencila Hub, with enhancements and new features driven by user engagement. The April 2019 update contains fixes to reliabilty and tweaked styles.
+---
+
+Stencila Hub is the web application that allows users to easily use Stencila's tools for executable documents without having to install them on their own machine. In addition, the Hub allows users to manage projects, accounts, teams, execution sessions and remote code and data sources. Since February we have mostly been working on invisible changes to improve reliability, but there have also been a few small but important style updates.
+
+
+## Style Updates
+
+Stencila's new Product Designer, Jacqueline, joined us in March, and quickly got to work on Stencila Hub. Her first task was to review and begin standardising some of its look and feel. Here's a quick before and after of the Projects List screen.
+
+![Before after Projects List](before-after.gif)
+
+You will have noticed other improvements to the consitency of buttons and general UX. There are more changes to come as we redesign some of the interface to support upcoming features, and we will announce them as they are released. But you will also see consistent incremental improvements to the UI until then.
+
+## Session Queueing
+
+When launching a Stencila Desktop session through Stencila Hub, there is a chance that an execution host is not ready and has to be started. This might take a couple of minutes – although we have work in the pipeline to reduce startup time in the future. While session queueing is not new to Stencila Hub, we have made a number of fixes and improvements to improve reliability.
```

Comment:
The double `improvement` stuck out a bit, very minor though.

```suggestion
When launching a Stencila Desktop session through Stencila Hub, there is a chance that an execution host is not ready and has to be started. This might take a couple of minutes – although we have work in the pipeline to reduce startup time in the future. While session queueing is not new to Stencila Hub, we have made a number of improvements to the reliability.
```